docs/guides/automating-a-process-using-bpmn.md

```diff
@@ -0,0 +1,102 @@
+
+Processes are the algorithms that determine how an organization runs based on independent tasks. Successful businesses grow from proven, effective processes. Therefore, Camunda’s workflow engine executes processes defined in BPMN.
+
+Take the following example where we've outlined a diagram to send an email. Don't worry too much about the symbols as we'll get to that shortly. For now, recognize the start and end of the process, comprised of entering a message, and sending the email.
+
+![sending email bmmn diagram](./img/simple-bpmn-process.png)
+
+Implement control and visibility over your critical business processes, as the workflow engine orchestrates processes that span APIs, microservices, business decisions and rules, human work, IoT devices, RPA bots, and more.
+
+## Set up
+
+At Camunda, we utilize [Modeler](./components/modeler/overview.md) to build out BPMN diagrams.
+
+You can begin building out your processes within Camunda Cloud using Cloud Modeler. Therefore, ensure you’ve [created a Camunda Cloud account](getting-started/create-camunda-cloud-account.md) to get started.
+
+## Getting started with BPMN
+
+Once logged in to your Camunda Cloud account, take the following steps:
+
+1. Click the **Diagrams** tab in the top navigation bar.
+2. Click **Create New Diagram**.
+3. Select the three vertical dots next to **New Diagram** on the top left corner of the page to rename your diagram. In this case, we'll name ours "Bake a Cake."
+
+### BPMN elements
+
+Before we build out the diagram to bake our cake, we need to understand what the components on the left side of the screen signify.
+
+We can build out our process using several elements, including the following:
+
+- Events: The things that happen. For example, start and end events which begin and terminate our process.
+- Tasks: For example, user tasks for a particular user to complete, or service tasks to invoke various webservices.
+- Gateways: For example, parallel gateways that move the process along between two tasks at the same time.
+- Subprocesses: For example, a transaction subprocess which can be used to group multiple activities to a transaction.
+
+For a complete list of BPMN elements, visit our [BPMN reference material](https://docs.camunda.org/manual/7.16/reference/bpmn20/).
+
+If you're new to BPMN elements and their capabilities, also visit our [BPMN symbol documentation](https://camunda.com/bpmn/reference/).
+
+### BPMN in action
+
+Using these elements, we can build out our process in baking a cake.
+
+Take the following steps:
+
+1. On our diagram, we've already been given an element as a start event in the shape of a circle. We can click on the circle, and then the wrench icon to adjust this element. For now, we'll keep it as a start event. Double click on the circle to add text.
```

Comment:
Here we talk about `we` sometimes and then switch to `you`. Consider to make consistent

Reply:
A downfall of mine -- thanks for calling it out! Updated in latest commit.